pkg/buildscript/buildscript.go

```diff
@@ -18,12 +18,20 @@ func New() (*BuildScript, error) {
 	return UnmarshalBuildExpression([]byte(emptyBuildExpression), nil)
 }
 
-func (b *BuildScript) AtTime() *time.Time {
-	return b.raw.AtTime
+func (b *BuildScript) Project() string {
+	return b.raw.CheckoutInfo.Project
+}
+
+func (b *BuildScript) SetProject(url string) {
+	b.raw.CheckoutInfo.Project = url
+}
+
+func (b *BuildScript) AtTime() time.Time {
+	return b.raw.CheckoutInfo.AtTime
 }
 
 func (b *BuildScript) SetAtTime(t time.Time) {
-	b.raw.AtTime = &t
+	b.raw.CheckoutInfo.AtTime = t
```

Comment:
I just realized; this probably shouldn't be on `raw` at all. `raw` covers the build expression portion, which this is not a part of.

That said I don't recall what entanglements exist. I'd suggest changing this to the buildscript (non-raw) package itself if it is trivial, otherwise we can get to it later.